contracts/truefi/TrueLender.sol

```diff
@@ -464,31 +464,20 @@ contract TrueLender is ITrueLender, Ownable {
     }
 
     /**
-     * @dev Check if a loan is within APY bounds
-     * Minimum absolute value of yes votes, rather than ratio of yes to no
-     * @param amount Size of loan
-     * @param yesVotes Number of yes votes
+     * @dev Check if a loan has enough votes to be approved
+     * @param votes Total number of votes
      * @return Whether a loan has reached the required voting threshold
      */
-    function votesThresholdReached(uint256 amount, uint256 yesVotes) public view returns (bool) {
-        return amount.mul(participationFactor) <= yesVotes.mul(10000).mul(TOKEN_PRECISION_DIFFERENCE);
+    function votesThresholdReached(uint256 votes) public view returns (bool) {
+        return votes >= minVotes;
     }
 
     /**
-     * @dev Use APY and term of loan to check expected value of a loan
-     * Expected value = profit - (default_loss * (no / yes))
-     * e.g. riskAversion = 10,000 => expected value of 1
-     * @param apy APY of loan
-     * @param term Term length of loan
+     * @dev Check if yes to no votes ratio reached the minimum rate
      * @param yesVotes Number of YES votes in credit market
      * @param noVotes Number of NO votes in credit market
      */
-    function loanIsCredible(
-        uint256 apy,
-        uint256 term,
-        uint256 yesVotes,
-        uint256 noVotes
-    ) public view returns (bool) {
-        return apy.mul(term).mul(yesVotes).div(365 days) >= noVotes.mul(riskAversion);
+    function loanIsCredible(uint256 yesVotes, uint256 noVotes) public view returns (bool) {
+        return yesVotes >= noVotes.mul(minYesToNoRatio);
```

Comment:
change logic to something like:

```
uint256 total = yes.add(no);
return yes >= total.mul(minRatio).div(BASIS_PRECISION);
```

The function is supposed to return true if the amount of yes votes is at least the minRatio of yes / total